docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# Contributing
+
+### Issues, Bugs, Feature or Component Requests
+
+While we currently don't accept issues from people outside the Adevinta Platform to this repo, we
+welcome [discussions and questions here](https://github.com/adevinta/spark-android/discussions/categories/general)!
+
+## If you're part of the Adevinta Platform ⬇️
+
+### Requirements
+
+#### [git-lfs](https://git-lfs.com/)
+
+This repository uses git-lfs to handle [cashapp/paparazzi](https://github.com/cashapp/paparazzi)
+screenshot testing.  
+If you introduce visible changes, you'll likely have to update screenshots files for the tests to
+pass.
+
+```bash
+# Install
+brew install git-lfs
+# Check
+git lfs install
+```
+
+#### [Java 17](https://github.com/adevinta/spark-android/issues/74)
+
+We currently use **AGP 8.1.0** which requires developers to use **JDK 17** on Gradle JDK.
+
+You can find the latest JDK 17
+release [here](https://www.azul.com/downloads/zulu-community/?version=java-17-lts&architecture=x86-64-bit&package=jdk).
+
+_If you're on macOS, you can install it with [brew](https://github.com/mdogan/homebrew-zulu/)_
+
+```bash
+brew tap mdogan/zulu
+brew install zulu-jdk17
+```
```

Comment:
IMO we should not present this information like this.

1. Zulu is one of many distributions, and we should not enforce anyone to use it, since we do not depend on it.
2. We should not provide a link to a custom brew tap. Instead, we should provide a better default like
   ```bash
   brew install openjdk@17
   ```
   and mention that M1 (apple silicon chip) devices might need a zulu distribution and link to their website instead (or [this](https://formulae.brew.sh/cask/zulu#default))